tests/cri-o/run-tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+cd $1
+
+if test "$(id -u)" != 0; then
+	echo "run as root"
+	exit 1
+fi
+
+(cd /crun; git clean -fdx; ./autogen.sh && ./configure CFLAGS='-Wall -Wextra -Werror' --prefix=/usr && make -j $(nproc) && make install)
+
+# make sure runc is not used
+rm -f /usr/bin/runc /usr/local/bin/runc
+ln /usr/bin/crun /usr/bin/runc
+
+rm -f /usr/bin/journalctl
+
+export CONTAINER_CGROUP_MANAGER=cgroupfs
+export RUNTIME_BINARY_PATH=/usr/bin/crun
+export CONTAINER_CONMON_CGROUP=pod
+export JOBS=2
+
+# adapt tests to cgroupfs
+sed -i 's/\.slice//g' test/testdata/*.json
+sed -i -e 's|@test "conmon custom cgroup" {|@test "conmon custom cgroup" {\nskip\n|g' test/*.bats
+sed -i -e 's|@test "privileged ctr device add" {|@test "privileged ctr device add" {\nskip\n|g' test/*.bats
+sed -i -e 's|@test "ctr device add" {|@test "ctr device add" {\nskip\n|g' test/*.bats
+sed -i -e 's|@test "privileged ctr -- check for rw mounts" {|@test "privileged ctr -- check for rw mounts" {\nskip\n|g' test/*.bats
+sed -i -e 's|@test "kubernetes pod terminationGracePeriod passthru" {|@test "kubernetes pod terminationGracePeriod passthru" {\nskip\n|g' test/*.bats
```

Comment:
Few tests are skipped, from last fail i was able to infer something related to `escalated privilege` not sure.
But should we add a reason for skipping these tests.

Reply:
they don't run in a container.

We could run them directly in the GH Actions, but I prefer to run them in a container so it is easier to run them locally (and less dependent on GH if we ever decide to move somewhere else in the future)

Reply:
fair point.